Lib/idlelib/NEWS.txt

```diff
@@ -3,6 +3,8 @@ Released on 2019-10-20?
 ======================================
 
 
+bpo-37902: Add scrolling for IDLE browsers.
```

Comment:
I appreciate your initiative, but this will become a merge conflict as soon as another news item is added by another PR.  So I add these last thing when ready to merge.  Please delete.

Reply:
No problem :)